src/NodeLink.cpp

```diff
@@ -93,7 +93,6 @@ namespace vizkit
         NodeCallback *callback = new NodeCallback(node1,node2);
         geode->setUpdateCallback(callback);
         object->addChild(geode);
-        
         return (::osg::Node*)object;
```

Comment:
The explicit cast should not be needed here, no ?

Reply:
True, but I didn't even change that line.